Documents/Institutions.md

```diff
@@ -0,0 +1,62 @@
+# Institutions Data Model
+
+This document describes the data model for Financial Institutions in HMDA. 
+A Financial Institution Database is created in HMDA for the purposes of 
+filing mortgage data. The information in this database is used in some edits, 
+as complementary information for publication and analysis of mortgage data, 
+and in the creation of the HMDA Panel. The HMDA Panel is the list of institutions
+that are required to file HMDA data, every year. 
+
+The Institutions Database needs to keep historical information for analysis, 
+and at least have information readily available for the two previous years in order
+to be able to perform the necessary checks when performing validation on the HMDA
+data being submitted or when composing the HDMA Panel. In addition to this, it must
+maintain the relationships between institutions (i.e. parent/child relationships)
+ and the actions that modify them (i.e. mergers, acquisitions). 
+ 
+ 
+The end product is a list of institutions that contains the necessary information 
+so that the HMDA Panel can be processed automatically. This table contains the following
+information:
+
+
+* `Activity Year` (integer length 4, i.e. 2017)
+* `Respondent ID` (string length 10)
```

Comment:
We should store history of Respondent ID to help with publication as part of the panel data. If an institution files in 2014 under one respondent id and agency code combo and then another in 2015 and then another in 2016 (maybe goes from being  a member bank to not being a member bank to being a member bank), we will need to be able to link all that history to track HMDA filings for historic reporting.